uaa-server/uaa.yml

```diff
@@ -33,14 +38,19 @@
       uaa.admin: Act as an administrator throughout the UAA
       uaa.none: Forbid acting as a user
       uaa.offline_token: Allow offline access
+      # ----- <Freddy's BBQ> ----- #
       order.me: Permission to read personal orders
       order.admin: Permission to read all orders
       menu.read: Permission to read all menu items
       menu.write: Permission to create, update, and delete menu items
+      # ----- <Freddy's BBQ> ----- #
+      sampleapi.read: Permission to access a specific endpoint in the Steeltoe App Security Samples
```

Comment:
I'm wondering if it would be helpful to add `sampleapi.write`. Not used today, but makes it possible to use multiple roles in our samples. And then change this line in the samples:
```c#
public const string UnknownJwtScope = "sampleapi.write";
```
to:
```c#
public const string UnknownJwtScope = "sampleapi.unknown";
```

Reply:
I'd be more inclined to add it if/when we have a use case.